src/LinAlg/hiopLinSolverIndefDenseMagma.cpp

```diff
@@ -397,4 +424,137 @@ namespace hiop
     return true;
   }
 
+#if 0
+  hiopLinSolverIndefDenseMagmaBuKa_old2::hiopLinSolverIndefDenseMagmaBuKa_old2(int n, hiopNlpFormulation* nlp_)
+    : hiopLinSolverIndefDense(n, nlp_), work_(NULL)
+  {
+    ipiv_ = new int[n];
+  }
```

Comment:
Do we need to keep commented out code here?

Reply:
will be removed later , but for now will keep it just in case we need to rollback parts of it.